addons/calendar/models/calendar.py

```diff
@@ -615,7 +615,9 @@ def todate(date):
             # Convert the event date to saved timezone (or context tz) as it'll
             # define the correct hour/day asked by the user to repeat for recurrence.
             event_date = event_date.astimezone(timezone)  # transform "+hh:mm" timezone
-            rset1 = rrule.rrulestr(str(self.rrule), dtstart=event_date, forceset=True, tzinfos={})
+            naive_localized = event_date.replace(tzinfo=None)
+            rset1 = rrule.rrulestr(str(self.rrule), dtstart=naive_localized, forceset=True, ignoretz=True)
```

Comment:
you could rewrite this without duplicating the line `rset1 = ...`. And maybe add a meaningful comment